themes/default/content/docs/concepts/vs/terraform/_index.md

```diff
@@ -25,7 +25,7 @@ aliases:
     }
 </style>
 
-Choosing the right infrastructure as code platform is critical. We want you to have the best information possible to pick the solution that fits your needs.
```

Comment:
in the What is TF section 2 para below, we hyperlink infrastructure as code. Can we move that hyperlink to this first instance instead?

Reply:
yup